stacks/ucan-invocation-stack.js

```diff
@@ -41,14 +43,16 @@ export function UcanInvocationStack({ stack, app }) {
       resources: [agentMessageBucket.cdk.bucket.arnForObjects('*')],
     })
   )
-
+  
   const agentIndexBucket = new Bucket(stack, 'invocation-store', {
     cors: true,
     cdk: {
       bucket: getBucketConfig('invocation-store', app.stage)
     }
   })
 
+  const agentIndexTable = new Table(stack, 'invocation-table', agentIndexTableProps)
```

Comment:
Minor, but it's an index of items in agent messages, so I'd probably go for `agent-index` or `agent-message-index` as the table name. Whilst `invocation-table` aligns with the name of the old bucket (`invocation-store`), we have long since moved on from calling this `invocation-store` in the (majority of the) code. `agent-index` aligns with what we're calling it in the code today.

Also, I don't think we need need to suffix with `-table` - all dynamo tables are tables.

```suggestion
  const agentIndexTable = new Table(stack, 'agent-index', agentIndexTableProps)
```